### PR TITLE
Design Login User Form

### PR DESCRIPTION
**Pull request by paradus-hex**

### CODE REVIEW

File: packages/client/components/pages/register/RegisterForm.tsx [hunk 1]
```diff
@@ -19,6 +20,18 @@ export default function RegisterForm({
   const { isLoading, mutateAsync } = useRegisterUserMutation();
   const registerUserMutationCacheFn = useRegisterUserMutationCache();
 
+  const LoginLink = styled(Typography)(({ theme }) => ({
```
Comment (Devorein) [inline packages/client/components/pages/register/RegisterForm.tsx:23]: Please move this outside the component

File: packages/client/components/pages/login/LoginForm.tsx [hunk 2]
```diff
@@ -23,67 +25,136 @@ export default function LoginForm({
...
-              </LoadingButton>
-            ) : (
-              <Button type='submit' disabled={isSubmitting || !isValid}>
-                {apiConstants.loginUser.submitButtonText}
-              </Button>
-            )}
-          </Form>
-        )}
-      </Formik>
+  const RegisterLink = styled(Typography)(({ theme }) => ({
```
Comment (Devorein) [inline packages/client/components/pages/login/LoginForm.tsx:28]: Please create a new component and reuse it inside LoginForm and RegisterForm

**Review by Devorein — CHANGES_REQUESTED** [review]

File: packages/client/components/pages/login/LoginForm.tsx (hunk 2 above)
Comment (paradus-hex) [reply packages/client/components/pages/login/LoginForm.tsx]: I am having some issues refactoring the styled component into a separate common component. Kindly check discord for more details.

**Review by paradus-hex — COMMENTED** [review]
LGTM

**Review by Devorein — APPROVED** [review]

**Pull request merged**